src/main/java/org/fundacionjala/coding/manuel/movies/NewReleaseMovie.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.coding.manuel.movies;
+
+/**
+ * Created by Administrator on 3/19/2018.
+ */
+public class NewReleaseMovie extends Movie {
+    private static final int THREE = 3;
+    /**
+     * Constructor.
+     * @param title .
+     * @param priceCode .
+     */
+    public NewReleaseMovie(String title, int priceCode) {
+        super(title, priceCode);
+    }
+
+    /**
+     * Method to calculate amount.
+     * @param daysRented .
+     * @return amount.
+     */
+    @Override
+    public double calculateAmount(int daysRented) {
+        return daysRented * THREE;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)